main.bal

```diff
@@ -306,7 +306,10 @@ service / on new http:Listener(port) {
             createdStep = check database:createTimelineStep(
                 experimentId = experimentId,
                 parameters = stepData.parameters,
-                processorName = stepData.processorName
+                parametersContentType = stepData.parametersContentType,
+                processorName = stepData.processorName,
+                processorVersion = stepData.processorVersion,
+                processorLocation = stepData.processorLocation
```

Comment:
Just for reference, a multiline comment looks like this (also a test for myself)

Reply:
Github did a very good job of making the difference clear... 🤦‍♂️

Reply:
Okay, good to know^^